### ISSUE TITLE
Excess whitespaces when formatting SwiftUI code

### ISSUE THREAD

**Issue by Rusik** [nicklockwood/SwiftFormat#987]
If initializer contains closures then formatter adds excess whitespaces in the beginning of next line with function call.
 
You can see example here
```swift
Button(
  title: "dasd"
)
.buttonStyle(AddToBagButtonStyle()) // <- This is correct

Button(action: {
  print("asdas")
})
  .buttonStyle(AddToBagButtonStyle()) // <- This is not

Button(action: {
  print("asdas")
}, label: {
  print("asdas")
})
  .buttonStyle(AddToBagButtonStyle()) // <- This is not
```

But it should looks like this

```swift
Button(
  title: "dasd"
)
.buttonStyle(AddToBagButtonStyle()) // <- This is correct

Button(action: {
  print("asdas")
})
.buttonStyle(AddToBagButtonStyle()) // <- This is correct too

Button(action: {
  print("asdas")
}, label: {
  print("asdas")
})
.buttonStyle(AddToBagButtonStyle()) // <- This is correct too
```

Is there any way to avoid this behaviour?

**Comment by nicklockwood** [comment]
@Rusik seems to be a bug. I'll look into it.

**Comment by timstudt** [comment]
I found similar bug after updating from `0.47.11` -> `0.48.11`. It adds extra spaces on next line after `))` double closing braces
<img width="467" alt="Screenshot 2021-09-13 at 12 12 05" src="https://user-images.githubusercontent.com/2987744/133067082-43bcc332-a36e-461e-8773-da74751e9a2c.png">

it seems to align with `(` of above line instead of 1 tab in from prev. line

**Comment by nicklockwood** [comment]
@Rusik fixed in 0.48.12
